testing/kubernetes/helloworld.yaml

```diff
@@ -86,13 +83,13 @@ spec:
     spec:
       containers:
       - name: helloworld
-        image: amalgam8/a8-${A8_TEST_ENV}-helloworld-v1:${A8_RELEASE}
+        image: amalgam8/a8-examples-helloworld-v1:latest
         imagePullPolicy: IfNotPresent
         ports:
         - containerPort: 5000
         env:
-        - name: A8_SERVICE
-          value: helloworld:version=v1
+        - name: A8_SERVICE_VERSION
+          value: v1
```

Comment:
The version format for docker (and previous k8s) was version=v1.  The testing script looks for that when validating.